Tribler/Core/Modules/restapi/search_endpoint.py

```diff
@@ -96,11 +100,16 @@ def render_GET(self, request):
             request.setResponseCode(http.BAD_REQUEST)
             return json.dumps({"error": "Trying to query for unknown type of metadata"})
 
+        search_uuid = self.get_uuid(request.args)
+        if not search_uuid:
```

Comment:
Can we make this optional? There are times when I'm using the REST API with curl and it is inconvenient if I have to specify a UUID every time.